daemon/daemon.go

```diff
@@ -853,7 +854,7 @@ func (fc *DataplaneConnector) handleProcessStatusUpdate(ctx context.Context, msg
 		FirstUpdate:   !fc.firstStatusReportSent,
 	}
 	kv := model.KVPair{
-		Key:   model.ActiveStatusReportKey{Hostname: fc.config.FelixHostname},
+		Key:   model.ActiveStatusReportKey{Hostname: fc.config.FelixHostname, RegionString: model.RegionString(fc.config.OpenstackRegion)},
```

Comment:
Do we have a test for this anywhere?  Would it be easy to add?

Reply:
We have ST for it, in calico-test.

I guess we could also add a Felix FV, but it feels like it's better to get onto the ST, given time pressure.